src/Umbraco.Web/Models/ContentEditing/ContentTypeCompositionDisplay.cs

```diff
@@ -44,6 +44,10 @@ public ContentTypeCompositionDisplay()
         [DataMember(Name = "compositeContentTypes")]
         public IEnumerable<string> CompositeContentTypes { get; set; }
         
+        //Locked compositions
+        [DataMember(Name = "lockedCompositeContentTypes")]
+        public IEnumerable<string> LockedCompositeContentTypes { get; set; }

```

Comment:
I don't think this will be required - we need to review/merge U4-7588 before we merge this in, there is some overlap and I think that what this value is being used for in this PR is superceded by what is in the other PR.